modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,38 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing statistics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect node network performance statistics and pass them to the `nmstate-metrics` API. You can enter a command on the OpenShift CLI (`oc)` to view these statistics.
+
+You can choose to export the generated statistics to a third-party application, such as the Prometheus Query Language (PromQL), where you can run queries to examine the statistics and view them on a plot.
```

Comment:
AFAIK operator already exports statistics to the OpenShift Monitoring. So you shouldn't need to deploy your own Prometheus, you should query the OCP directly